src/data/spotifyDAO.ts

```diff
@@ -35,21 +38,21 @@ export class SpotifyAPI {
     public searchAlbums(searchTerm: string): Promise<SpotifyAlbumInfo[]> {
         return new Promise((resolve, reject) => {
             this.api.searchAlbums(searchTerm, {limit: 5})
-                .then((data) => {
-                    const albums = data.body.albums.items.map((item) => {
-                        return {
-                            id: item.id,
-                            name: item.name,
-                            // artist: item.artists[0].name,
-                            image: item.images[2].url,
-                            url: item.external_urls.spotify,
-                        };
-                    });
-                    resolve(albums);
-                }, (err) => {
-                    console.error(err);
-                    reject(err);
+            .then((data) => {
+                const albums = data.body.albums.items.map((item) => {
+                    return {
+                        id: item.id,
+                        name: item.name,
+                        // artist: item.artists[0].name,
```

Comment:
Meant to be commented out?

Reply:
yup. That type is broken and needs to be fixed. The object returned by Spotify has the `artist` field, but the type doesn't define it, so it won't build.